tests/integration/services.test.js

```diff
@@ -38,40 +42,67 @@ describe.skip('UserMergeService', function () {
 
       await UserMergeService.mergeAccounts(ogUser1, ogUser2)
 
-      // TODO: get from db
-      const newParent = ogUser2
-      const [newChild1, newChild2, newChild3] = [ogUser1, mergedUser1, mergedUser2]
+      const newParent = getUser(ogUser2)
```

Comment:
This allows `ogUser2`'s userId to change? Why not just set keep `      const newParent = ogUser2`?

Reply:
Because were checking for `newParent.mergeduserid` to be null so we need to get the user from the db.